Jenkinsfile

```diff
@@ -88,7 +91,16 @@ pipeline {
         success {
           script {
             if (params.RELEASE_BUILD && !params.RELEASE_DRY_RUN) {
```

Comment:
What happens in the case of a hotfix?

Reply:
We don't currently support hotfixes for soql-postgres-adapter